tracing/lobster_codebeamer/codebeamer.test-specifications.lobster-trlc.yaml

```
@@ -0,0 +1,6 @@
+inputs:
+  - "lobster/use_cases.trlc"
+  - "lobster/requirements.rsl"
+  - "lobster/tools/codebeamer/requirements/potential_errors.trlc"
+  - "lobster/tools/codebeamer/requirements/test_specifications.trlc"
```

Just specifying the requirements folder path should suffice.
```suggestion
  - "lobster/tools/codebeamer/requirements/"
```